### ISSUE TITLE
`podman {FOO}` results in "cannot open: /proc/$NONEXISTENT_PID/ns/user" on fresh F32 install

### ISSUE THREAD

**Issue by ben-alkov** [containers/libpod#6098]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**

`podman {FOO}` results in "cannot open: /proc/$NONEXISTENT_PID/ns/user" on fresh F32 install

Typing ANY podman command (except `version`) results in "cannot open: /proc/$PID/ns/user" where $PID doesn't exist in /proc.

**Steps to reproduce the issue:**

1. Install Fedora 32 release (KDE spin, if that matters) from scratch - nothing on the hard drive.
1. Log in
1. Try to run any podman command

**Describe the results you received:**

"cannot open: /proc/$PID/ns/user"

**Describe the results you expected:**

podman works

**Output of `podman version`:**

```text
Version:            1.9.1
RemoteAPI Version:  1
Go Version:         go1.14.2
OS/Arch:            linux/amd64
```

**Output of `podman info --debug`:**

```
cannot open: /proc/1995/ns/user
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
podman-1.9.1-1.fc32.x86_64
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

Bare metal on a Lenovo Thinkpad laptop

**Comment by mheon** [comment]
@giuseppe PTAL - looks like a pause process issue, but still happening on v1.9.1 (and a fresh install, at that)

**Comment by ben-alkov** [comment]
I don't understand how this could have happened, but I had **neither** `/etc/subuid` **nor** `/etc/subgid`.

Simply following the ~~prompts~~ errors from running `podman system reset` (as NON-root) led me to `man 5 subuid` and creating the appropriate files.

**Comment by summatix** [comment]
I'm now getting this same error

**Comment by mheon** [comment]
Podman version? Fedora release?

**Comment by summatix** [comment]
Solution for me was:

    rm -rf /tmp/run-`id -u`

**Comment by summatix** [comment]
> 
> 
> Podman version? Fedora release?

Fedora 32, podman 1.9.1

**Comment by mkoura** [comment]
I'm getting the same error on Fedora 31 with podman 1.9.1

**Comment by mkoura** [comment]
@mheon I cannot reproduce the issue on v1.9.2. Thanks!

**Comment by summatix** [comment]
I just had this error happen again. Maybe I need to make a habit of running `rm -rf /tmp/run-$(id -u)` whenever I want to use podman.

**Comment by mheon** [comment]
What Podman version? 1.9.2 doesn't appear to be in stable yet, but it should be landing there tomorrow (it's marked for stable as of today).

**Comment by summatix** [comment]
I'm still using 1.9.1. It's a simple enough problem to fix so I'll wait for 1.9.2 to be released to stable. The trouble is I don't know how to reproduce the error, it just happens sometimes.

**Comment by giuseppe** [comment]
> I'm still using 1.9.1. It's a simple enough problem to fix so I'll wait for 1.9.2 to be released to stable. The trouble is I don't know how to reproduce the error, it just happens sometimes.

I think the easiest reproducer is to run `killall podman` as rootless and it will also kill the pause process.

**Comment by rhatdan** [comment]
dnf -y update --enablerepo=update-testing podman

And you will get the new version.
